shared/constants/network.js

```diff
@@ -309,3 +315,86 @@ export const BUYABLE_CHAINS_MAP = {
     },
   },
 };
+
+export const FEATURED_RPCS = [
+  {
+    chainId: '0xA4B1',
+    nickname: 'Arbitrum One',
+    rpcUrl: `https://arbitrum-mainnet.infura.io/v3/${infuraProjectId}`,
+    ticker: 'AETH',
+    rpcPrefs: {
+      blockExplorerUrl: 'https://explorer.arbitrum.io',
+      imageUrl: AETH_TOKEN_IMAGE_URL,
+    },
+  },
+  {
+    chainId: '0xA86A',
```

Comment:
I think we will want to reference the same constants where possible. We have defined constants at the top of this file

https://github.com/MetaMask/metamask-extension/blob/9dc94f99bea46acd888658b2e5e9ca6a3d30e438/shared/constants/network.js#L24-L30

Similarly there is also tickers and nicknames which should remain consistent across the app
https://github.com/MetaMask/metamask-extension/blob/9dc94f99bea46acd888658b2e5e9ca6a3d30e438/shared/constants/network.js#L44-L46

Could we follow this convention? Then I think the network image will show up here.

<img width="984" alt="Screen Shot 2022-06-01 at 3 26 15 PM" src="https://user-images.githubusercontent.com/8112138/171512027-7e99a238-bbdc-435e-a5e6-ba18137f9b7f.png">

Reply:
Fixed: f0c4ca1d42e706eb710e1c2d54578a354d2de0ee

<img width="981" alt="Screenshot 2022-06-02 at 13 01 45" src="https://user-images.githubusercontent.com/92310504/171617367-00a2d0fa-9225-4c69-ba1b-060bd139634a.png">